components/devtools/actors/preference.rs

```diff
@@ -37,7 +38,13 @@ impl Actor for PreferenceActor {
         stream: &mut TcpStream,
         _id: StreamId,
     ) -> Result<ActorMessageStatus, ()> {
-        let mut key = msg.get("value").unwrap().as_str().unwrap();
+        let mut key = match msg.get("value").unwrap() {
+            Value::String(value) => value.as_str(),
+            _ => {
+                warn!("PreferenceActor: handle_message: value is not a string");
+                return Ok(ActorMessageStatus::Ignored);
+            },
+        };
```

Comment:
Perhaps this could be a bit simpler?

```suggestion
        let mut Some(key) = match msg.get("value").unwrap().as_str() else {
            warn!("PreferenceActor: handle_message: value is not a string");
            return Ok(ActorMessageStatus::Ignored);
        };
```

Reply:
I tried it, but doesn’t it seem like the correct syntax?

Reply:
Yeah, the match keyword needs to be removed.